examples/tags-in-searchbox/app.tsx

```diff
@@ -0,0 +1,366 @@
+import { createTagsPlugin, Tag } from '@algolia/autocomplete-plugin-tags';
+import algoliasearch from 'algoliasearch';
+import { h, Fragment, render } from 'preact';
+import groupBy from 'ramda/src/groupBy';
+import insightsClient from 'search-insights';
+
+import '@algolia/autocomplete-theme-classic';
+
+import { ProductHit, TagExtraData } from './types';
+
+const appId = 'latency';
+const apiKey = '6be0576ff61c053d5f9a3225e2a90f76';
+const searchClient = algoliasearch(appId, apiKey);
+
+// @ts-expect-error type error in search-insights
+insightsClient('init', { appId, apiKey });
+
+const algoliaInsightsPlugin = createAlgoliaInsightsPlugin({ insightsClient });
+
+const tagsPlugin = createTagsPlugin<TagExtraData>({
+  getTagsSubscribers() {
+    return [
+      {
+        sourceId: 'brands',
+        getTag({ item }) {
+          return item;
+        },
+      },
+      {
+        sourceId: 'categories',
+        getTag({ item }) {
+          return item;
+        },
+      },
+    ];
+  },
+  transformSource() {
+    return undefined;
+  },
+  onChange({ tags }) {
+    requestAnimationFrame(() => {
+      const container = document.querySelector('.aa-InputWrapperPrefix');
+      const oldTagsContainer = document.querySelector('.aa-Tags');
+
+      const tagsContainer = document.createElement('div');
+      tagsContainer.classList.add('aa-Tags');
+
+      render(
+        <div className="aa-TagsList">
+          {tags.map((tag) => (
+            <TagItem key={tag.label} {...tag} />
+          ))}
+        </div>,
+        tagsContainer
+      );
+
+      if (oldTagsContainer) {
+        container.removeChild(oldTagsContainer);
+      }
+
+      container.appendChild(tagsContainer);
+    });
+  },
+});
+
+type TagItemProps<TTag> = Tag<TTag>;
+
+function TagItem<TTag>({ label, remove }: TagItemProps<TTag>) {
+  return (
+    <div className="aa-Tag">
+      <span className="aa-TagLabel">{label}</span>
+      <button
+        className="aa-TagRemoveButton"
+        onClick={() => remove()}
+        title="Remove this tag"
+      >
+        <svg
+          fill="none"
+          stroke="currentColor"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          strokeWidth={2}
+          viewBox="0 0 24 24"
+        >
+          <path d="M18 6L6 18"></path>
+          <path d="M6 6L18 18"></path>
+        </svg>
+      </button>
+    </div>
+  );
+}
+
+autocomplete<ProductHit | Tag<TagExtraData>>({
+  container: '#autocomplete',
+  placeholder: 'Search',
+  openOnFocus: true,
+  plugins: [algoliaInsightsPlugin, tagsPlugin],
+  detachedMediaQuery: 'none',
+  getSources({ query, state }) {
+    const tagsByFacet = groupBy<Tag<TagExtraData>>(
+      (tag) => tag.facet,
+      state.context.tagsPlugin.tags
+    );
+
+    return [
+      {
+        sourceId: 'brands',
+        onSelect({ item, state, setQuery }) {
+          if (item.label.toLowerCase().includes(state.query.toLowerCase())) {
```

Comment:
If you type `samsung ` (notice the space), the query isn't reset.

This should work:

```suggestion
          if (item.label.toLowerCase().includes(state.query.toLowerCase().trim())) {
```